learn-arai/app/components/icons/google.tsx

```diff
@@ -0,0 +1,37 @@
+export const GoogleIcon = () => {
```

Comment:
Is there any reasons to use arrow function instead of regular function?
If there is only one export, consider default export this function

Reply:
done